### PR TITLE
BLOC-40 Fix moderation page

### PR DESCRIPTION
**Pull request by almondy22**

### CODE REVIEW

File: app/containers/Moderation/Content.jsx [hunk 1]
```diff
@@ -92,44 +90,62 @@ const Section = ({
...
   return (
     <SectionStyled id={sectionId}>
-      <BaseTransparent css={{ padding: '32px 32px 16px' }}>
+      <BaseTransparent
+        css={css`
+          padding: 32px 32px 16px;
+          @media only screen and (max-width: 576px) {
+            padding-left: 5px;
+          }
+        `}
```
Comment (AweAwe95) [inline app/containers/Moderation/Content.jsx:99]: Move all CSS to a styled.ts file

File: app/containers/Moderation/Content.jsx [hunk 2]
```diff
@@ -92,44 +90,62 @@ const Section = ({
...
+        <div className="d-block" key={h3}>
+          <div
+            css={css`
+              padding: 16px 32px;
+              font-size: 20px;
+              font-weight: 600;
+              @media only screen and (max-width: 576px) {
+                padding-left: 5px;
+              }
+            `}
```
Comment (AweAwe95) [inline app/containers/Moderation/Content.jsx:116]: Move all CSS to a styled.ts file

File: app/containers/Moderation/Content.jsx (hunk 1 above)
Comment (almondy22) [reply app/containers/Moderation/Content.jsx]: Done

File: app/containers/Moderation/Content.jsx (hunk 2 above)
Comment (almondy22) [reply app/containers/Moderation/Content.jsx]: Done

**Review by AweAwe95 — APPROVED** [review]

**Review by YuriyLu — APPROVED** [review]

**Pull request merged**